source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+  <li>Minimally validate that it conforms to the specification</li>
+  <li>Find the set of updates of the page at <code class="highlighter-rouge">page.orderedItems</code> (<code class="highlighter-rouge">items</code>)</li>
+  <li>In reverse order, iterate through the activities (<code class="highlighter-rouge">activity</code>) in <code class="highlighter-rouge">items</code>:
+    <ol>
+      <li>For each <code class="highlighter-rouge">activity</code>, if <code class="highlighter-rouge">activity.endTime</code> is before <code class="highlighter-rouge">lastCrawl</code>, then terminate ;</li>
+      <li>If the updated resource's uri at <code class="highlighter-rouge">activity.object.id</code> is in <code class="highlighter-rouge">processedItems</code>, then continue ;</li>
+      <li>Otherwise, if <code class="highlighter-rouge">activity.type</code> is <code class="highlighter-rouge">Update</code> or <code class="highlighter-rouge">Create</code>, then find the URI of the updated resource at <code class="highlighter-rouge">activity.object.id</code> (<code class="highlighter-rouge">object</code>) and process the resource ;</li>
+      <li>Otherwise, if <code class="highlighter-rouge">activity.type</code> is <code class="highlighter-rouge">Delete</code>, then find the URI of the deleted resource at <code class="highlighter-rouge">activity.object.id</code> and process its removal.</li>
+      <li>Add the processed resource's URI to <code class="highlighter-rouge">processedItems</code></li>
+    </ol>
+  </li>
+  <li>Finally, find the URI of the previous page at <code class="highlighter-rouge">collection.prev.id</code> (<code class="highlighter-rouge">pageN1</code>)</li>
+ <li>If there is a previous page, apply the results of the page algorithm to <code class="highlighter-rouge">pageN1</code></li>
+</ol>
+
+#### 3.5.3. Indexing
+{: #indexing}
+
+If the objective of the consuming application is to find descriptive information that might be used to build an index allowing the resources to be discovered, then the application _SHOULD_ use the IIIF Presentation API `seeAlso` property to discover an appropriate, machine-readable description of the resource.  For different types of resource, and for different domains, the external resources will have different formats and semantics. If there are no external descriptions, or none that can be processed, the data in the Manifest and in other IIIF resources might be used as a last resort, despite its presentational intent. 
+
+## 4. Network Considerations
+
+### 4.1. Activities for Access-Restricted Content
+
+Activities _MAY_ be published about content that has access restrictions. Clients _MUST NOT_ assume that they will be able to access every resource that is the object of an Activity, and _MUST NOT_ assume that if they cannot access it, then it has been permanently deleted.  The content might be protected by some authentication system, or there might simply be a temporary network outage. 
```

Comment:
Do we mean "permanently deleted" or simply "deleted"?

Reply:
The second sentence is the important one. The final sentence is intended as examples for when content might be inaccessible but not gone.  Added For example and messed a bit with the sentence.

Reply:
But I think the point is simply that you can't take it that the resource has been deleted -- permanently, temporarily, or otherwise.

Reply:
Oh, I see. Sure... the "permanently" was supposed to give some justification to why the crawler might think to delete it from the index, but I can just delete (ha) the word.

Reply:
:+1: